src/rpc/server/lib.zig

```diff
@@ -0,0 +1,11 @@
+const server = @import("server.zig");
+
+pub const Context = server.Context;
+pub const basic = server.basic;
+pub const LinuxIoUring = server.LinuxIoUring;
+
+comptime {
+    _ = Context;
+    _ = basic;
+    _ = LinuxIoUring;
+}
```

Comment:
whats going on here

Reply:
These are references to force analysis of the namespaces; otherwise tests don't run.

Reply:
can we add a note about that - i dont think thats the case for any other struct/files in the project so not sure why this is the case here 🤔

Reply:
Actually I just realized; the reason the tests aren't being picked up is because we're relying on the `refAllDeclsRecursive` function, which can only access public declarations, which `server` is not, which is where the main test is actually defined - it also works just by referencing `_ = server;` directly.

Reply:
Amended to this: 1e042dd9711bf93019c24cf95a90ba5046090762

Reply:
This is an annoying thing about zig. I can't figure out a good way to ensure every test is run. I'd like to just say once "please run every single test in the src folder" and zig figures out how to do that. But it doesn't work like that. Instead you need a bunch of weird hacks to make that happen.

Reply:
For what it's worth, they are working on a solution for this. I'll see if I can find the relevant issue or discussion.